gui/src/app/CompilationServerConnectionControl/ConfigureCompilationServerDialog.tsx

```diff
@@ -20,103 +29,103 @@ const ConfigureCompilationServerDialog: FunctionComponent<
-        <label htmlFor="local">Local server</label>
-        <br />
+      <FormControl>
+        <FormLabel id="compilation-server-selection">
+          Compilation server
+        </FormLabel>
+        <RadioGroup value={choice} onChange={makeChoice}>
+          <FormControlLabel
+            value="public"
+            control={<Radio />}
+            label="Public server"
+          />
+          <FormControlLabel
+            value="local"
+            control={<Radio />}
+            label="Local server"
+          />
+          <FormControlLabel
+            value="custom"
+            control={<Radio />}
+            label="Custom server"
+          />
+        </RadioGroup>
 
-        <input
-          type="radio"
-          id="custom"
-          name="server"
-          value="custom"
-          checked={choice === "custom"}
-          onChange={() => setChoice("custom")}
-        />
-        <label htmlFor="custom">Custom server</label>
-        <br />
-
-        <input
-          // This one isn't honoring a class-based style for some reason
-          style={{ width: 500 }}
-          disabled={choice !== "custom"}
-          type="text"
-          value={stanWasmServerUrl}
-          onChange={(e) => setStanWasmServerUrl(e.target.value)}
-        />
-        <br />
-        <hr />
-        {choice === "local" && (
-          <div>
-            <p>To start a local compilation server:</p>
-            <div>
-              <pre>
-                docker run -p 8083:8080 -it magland/stan-wasm-server:latest
-              </pre>
-            </div>
-          </div>
-        )}
-        {choice === "public" && (
+        {choice === "custom" && (
           <div>
             <p>
-              The public server is provided for convenience, but may not be as
-              reliable as a local server, depending on the current load and
-              availability.
+              <TextField
+                variant="standard"
+                label="Custom server URL"
+                disabled={choice !== "custom"}
+                value={stanWasmServerUrl}
+                onChange={(e) => setStanWasmServerUrl(e.target.value)}
+              />
             </p>
           </div>
         )}
-        <hr />
-        <Button onClick={() => closeDialog()}>Close</Button>
-      </div>
+      </FormControl>
+
+      {choice === "local" && (
+        <div>
+          <p>To start a local compilation server:</p>
+          <div>
+            <pre>
+              docker run -p 8083:8080 -it magland/stan-wasm-server:latest
+            </pre>
+          </div>
+        </div>
+      )}
+      {choice === "public" && (
+        <div>
+          <p>
+            The public server is provided for convenience, but may not be as
+            reliable as a local server, depending on the current load and
+            availability.
+          </p>
+        </div>
+      )}
+
+      <Divider />
+      <Button onClick={() => closeDialog()}>Close</Button>
```

Comment:
Wonder if we really need this, given that there's the `X` and it'll close if you click outside the dialog?

Reply:
Agreed, removed